simcon_project/users/views/researcher_home.py

```diff
@@ -1,7 +1,22 @@
 from django.shortcuts import render
+from django.db.models import Q
 from conversation_templates.models import TemplateResponse
 from django.contrib.auth.decorators import user_passes_test
-from django.core.paginator import Paginator
+import django_tables2 as tables
+from django_tables2 import RequestConfig
+
+
+class ResponseTable(tables.Table):
+    name = tables.columns.Column(accessor="student.get_full_name")
+    response = tables.TemplateColumn(
+        '''<a class="btn btn-info btn-sm" href="{% url 'view-response' record.id %}">View</a>''')
+    delete = tables.TemplateColumn(
+        '''<a class="btn btn-danger" href="{% url 'delete-response' record.id %}"  >Delete</a>''')
```

Comment:
We should either add a verbose name and set it to an empty string, TemplateColumn(verbosename='') and not have a column header or set it to TemplateColumn(orderable=False) to retain the header but have it not be clickable since it results in an error screen right now if the column header is clicked for the template columns.